versions/run.ver

```diff
@@ -28,7 +28,6 @@ export envvar_ver=1.0
 
 export gfs_ver=v16.3
 export gefs_ver=v12.3
-export aqm_ver=v7.0.8
```

Comment:
did you mean to remove this?

Reply:
Yes,  I do not see this line in the operational node.  We have done several tests to deal with the failure cases on Nov. 20-21, 2024 and today 00z cycle and the results are identical to operational forecast